src/Microsoft.DotNet.Build.Tasks/PackageFiles/init-tools.sh

```diff
@@ -13,13 +13,13 @@ if [ ! -d "$__PROJECT_DIR" ]; then
    exit 1
 fi
 
-if [ ! -d "$__TOOLS_DIR" ]; then
-   echo "ERROR: Cannot find tools path at '$__TOOLS_DIR'. Please pass in the tools directory as the 2nd parameter."
+if [ ! -e "$__DOTNET_CMD" ]; then
+   echo "ERROR: Cannot find dotnet.exe at path '$__DOTNET_CMD'. Please pass in the path to dotnet.exe as the 2nd parameter."
    exit 1
 fi
 
-if [ ! -e "$__DOTNET_CMD" ]; then
-   echo "ERROR: Cannot find dotnet.exe at path '$__DOTNET_CMD'. Please pass in the path to dotnet.exe as the 3rd parameter."
+if [ ! -d "$__TOOLRUNTIME_DIR" ]; then
```

Comment:
Most repos will probably mimic what we have in corefx where, as you mentioned previously, we create the toolRuntime dir before calling this script. However, I don't think this to be a requirement, since you could totally select a different folder as your toolruntime folder, and if it doesn't exist, then dotnet publish will create it for you, so I don't think that we should make it a requirement for the folder to already exist. This is why previous logic wasn't checking if the TOOLRUNTIME_DIR existed.